packages/expo-audio/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Update docs and api so all units of time should be in milliseconds, not seconds. ([#33320](https://github.com/expo/expo/pull/33320) by [@alanjhughes](https://github.com/alanjhughes))
```

Comment:
To be honest I have no idea how to qualify this change. Isn't it breaking?
```suggestion
- Update docs and API so all units of time are in seconds, not milliseconds. ([#33320](https://github.com/expo/expo/pull/33320) by [@alanjhughes](https://github.com/alanjhughes))
```

Reply:
Not really. It fixes an error. The docs said it should be seconds but  I was returning milliseconds